domains/certificates/Query.API/API/Models/Certificate.cs

```diff
@@ -0,0 +1,24 @@
+namespace API.Models;
+
+public class Certificate
+{
+    /// <summary>
+    /// Start timestamp for the certificate in Unix time
+    /// </summary>
+    public long Start { get; set; }
+
+    /// <summary>
+    /// End timestamp for the certificate in Unix time
+    /// </summary>
+    public long End { get; set; }
+
+    /// <summary>
+    /// Amount of energy measured in Wh
+    /// </summary>
+    public int Amount { get; set; }
+
+    /// <summary>
+    /// Id for the metering point
+    /// </summary>
+    public string MeteringPointId { get; set; }
```

Comment:
We have used other naming on the measurements and emissions api, i would recommend we use the same wording:
Also Long for quantity, since we have previously have hit the limit of an int.
https://github.com/Energinet-DataHub/energy-origin/blob/main/domains/measurements/Measurements.API/API/Models/Measurement.cs


    string GSRN,
    long DateFrom,
    long DateTo,
    long Quantity

Reply:
Regarding long for quantity, I have a question and would like your input: I considered long, but when we hit the limit that was related to adding many measurements together. Since this is a single certificate for a single metering point with a short time period, the quantity is not going to be above 2,147,483,647 Wh (+2 TWh). I do not mind using long in this case, so it is consistent with the remaining API, but in this case with a dedicated read/query model we can use a smaller data type. So do we want API consistency or smaller data footprint?

Regarding usage of GSRN: Do we want to keep calling the metering point id for "GSRN"? I think I heard something about moving away from calling it "GSRN" as it is a standard used in Denmark and not many other countries.

Reply:
Regarding long, I would agree, but we will most likely create a API bucketed on time, where it would be long, and to remove possible confusion about when to use int and when to use long in the future, json does not have the idea of longs, so it does not change anything there, it is only internally related.

GSRN is a global standard where we have bought a series: https://www.gs1si.org/Portals/0/GS1_Dokumentacija/GS1_Publikacije/Brosure_GS1_ANG/Introduction_GSRN.pdf so it is not a danish thing.